arches/app/models/tile.py

```diff
@@ -287,7 +287,7 @@ def check_for_missing_nodes(self):
                 node = models.Node.objects.get(nodeid=nodeid)
                 datatype = self.datatype_factory.get_instance(node.datatype)
                 datatype.clean(self, nodeid)
-                if self.data[nodeid] is None and node.isrequired is True:
+                if not self.data[nodeid] and node.isrequired is True:
```

Comment:
I found while testing this that if the boolean datatype is required and a user tries to set the value to `False`, the value will be reported as missing and fail to save. I looked into it a bit and I think what we need is a 'clean' method on the datatype similar to what is used on the string datatype: https://github.com/archesproject/arches/blob/e62ade0409c7e888dd31443d3411c6b3a70f7a79/arches/app/datatypes/datatypes.py#L101

Reply:
Thanks, that makes more sense.